### PR TITLE
Collections

### PR DESCRIPTION
**Pull request by OleksandrZheleznyi**

### CODE REVIEW

File: Collections/1/MyList.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Collections
+{
+    internal class MyList<T> : IEnumerable<T>
+    {
+        private T[] _value = null;
```
Comment (OleksandrIvanovTheBest) [inline Collections/1/MyList.cs]: values

File: Collections/2/MyDictionary.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        }
+
+        private void IncreaseArrayLength(int item = 1)
+        {
+            Array.Resize(ref _keyValuePairs, Count + item);
+        }
+
+        public IEnumerator<KeyValuePair<TKey, TValue>> GetEnumerator()
+        {
+            return ((IEnumerable<KeyValuePair<TKey, TValue>>)_keyValuePairs).GetEnumerator();
```
Comment (OleksandrIvanovTheBest) [inline Collections/2/MyDictionary.cs:69]: It's OK

File: Collections/2/MyDictionary.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        //Or use yield
+        /*
+        int position = -1;
+
+        public void Reset()
+        {
+            position = -1;
+        }
+
+        public IEnumerator<KeyValuePair<TKey, TValue>> GetEnumerator()
```
Comment (OleksandrIvanovTheBest) [inline Collections/2/MyDictionary.cs]: Remove this comented text

**Review by OleksandrIvanovTheBest — APPROVED** [review]

**Review by OleksandrIvanovTheBest — CHANGES_REQUESTED** [review]
Where is an additional task?

**Comment by OleksandrZheleznyi** [comment]
Where is an additional task? - additional task in Program.cs in the end of file - #4(small function)

**Review by OleksandrIvanovTheBest — APPROVED** [review]

**Pull request merged**